### ISSUE TITLE
Import to TS project fails

### ISSUE THREAD

**Issue by Aliaksandr-Kasko-JazzTeam** [facundoolano/google-play-scraper#659]
* Operating System: Ubuntu 22.04.3 LTS
* Node version: 18.17.1
* google-play-scraper version: 10.0.0
* TypeScript version: 5.2.2

### Description: 
Please sync `index.d.ts` with `index.js`. Because of import issue.

### Example code:
```ts
import gplay from "google-play-scraper";
```

### Error message:

```
TS1192: Module '"/path/to/project/node_modules/google-play-scraper/index"' has no default export.
```

If I import with
```ts
import * as gplay from "google-play-scraper";
```
and call, for example, list
```ts
gplay.list({country: "us"}).then(console.log, console.warn)
```
then I got an error
```
TypeError: gplay.list is not a function
```

Only one workaround exists for now is to suppress `import gplay from "google-play-scraper"` with `// @ts-ignore`

**Comment by jbigman** [comment]
It works perfectly on my side:

- Operating System: windows 11 Family
- Node version: 18.17.0
- google-play-scraper version: 10.0.0
- TypeScript version: 5.2.0


google-play-scraper/index.d.ts, line 320 :
```ts
export const list: IFnList
```

Usage :
```ts
import gplay from "google-play-scraper"

public async searchApps (name: string, lang: string, country: string): Promise<void> {
  const loadedGames = await gplay.search({ term: name, lang, country, fullDetail: true })
}
```

**Comment by Aliaksandr-Kasko-JazzTeam** [comment]
Oh, thank you @jbigman. I added these changes to my tsconfig:
```
"module":  "NodeNext" -> "ESNext",
"moduleResolution": "NodeNext" -> "Node"
```
and all starts working for me!
I think there should be some setup manuals.

Offtop: just a small changes to iteration over enums, but these are trifles.

**Comment by simon-abbott** [comment]
I have an actual fix for this issue in #702.
